plugins/Essentials/config.yml

```diff
@@ -715,7 +715,7 @@ newbies:
   # If so, use this format, replacing {DISPLAYNAME} with the player name.
   # If not, set to ''
   #announce-format: ''
-  announce-format: '&dWelcome {DISPLAYNAME}&d to the server!'
+  announce-format: '&7Welcome {DISPLAYNAME}&7 to the &6&lSpigot&e&lCraft&7!'
```

Comment:
@NexusHD - as pangeacake stated, please remove "the."

You could just do a search for SpigotCraft or something (or clone and edit yourself), but I'm not really in favor of creating a "template" server. It's best that those who start a server take their time and learn about the plugins they are adding/using.